server/src/main/xsd/gateway-config-201409.xsd

```diff
@@ -1516,6 +1516,114 @@
         </complexContent>
     </complexType>
 
+    <complexType name="httpProxyServicePropertiesType">
+        <complexContent>
+            <restriction base="gateway:ServicePropertiesType">
+                <sequence>
```

Comment:
I'm ok with free form and would prefer that to having a strict sequence given the low quality exception that is thrown if you get the order wrong.